src/GuppyDataExplorer/ExplorerFilter/index.jsx

```diff
@@ -120,7 +120,11 @@ class ExplorerFilter extends React.Component {
   render() {
     const filterProps = {
       filterConfig: this.props.filterConfig,
-      guppyConfig: { type: this.props.guppyConfig.dataType, ...this.props.guppyConfig },
+      guppyConfig: {
+        type: this.props.guppyConfig.dataType,
+        ...this.props.guppyConfig,
+        explorerHideDataFilter,
```

Comment:
Don't alter the `guppyConfig` object in here to pass this, since this object represents the `guppyConfig` object in portal config for each individual `explorerConfig` section

Just pass this new option as a seprate argument